ic-os/components/hostos-scripts/misc/fetch-mgmt-mac.sh

```diff
@@ -67,12 +57,13 @@ function fetch_mgmt_mac() {
 
 function main() {
     # Establish run order
-    mgmt_mac=""
-    read_variables
-    if [ "${mgmt_mac}" == "" ]; then
+
+    MGMT_MAC=$(jq -r ".deployment.mgmt_mac" ${DEPLOYMENT})
```

Comment:
Could you please add that same comment about why we use `jq` here?

Reply:
Added!